From 6b38aa6278cb8e622fbb507407a6706501426be5 Mon Sep 17 00:00:00 2001
From: Uwe Klotz <uwe_klotz@web.de>
Date: Sat, 9 Nov 2013 09:31:44 +0100
Subject: [PATCH] Improve detection of libmp4/libmp4v2

This was needed to compile Mixxx with faad enabled on Fedora 19
---
 build/features.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/build/features.py b/build/features.py
index 72dd7dd..d324e1c 100644
--- a/build/features.py
+++ b/build/features.py
@@ -523,10 +523,11 @@ class FAAD(Feature):
 
         have_mp4v2_h = conf.CheckHeader('mp4v2/mp4v2.h')
         have_mp4v2 = conf.CheckLib(['mp4v2','libmp4v2'], autoadd=False)
+        have_mp4_h = conf.CheckHeader('mp4.h')
         have_mp4 = conf.CheckLib('mp4', autoadd=False)
 
         # Either mp4 or mp4v2 works
-        have_mp4 = (have_mp4v2_h and have_mp4v2) or have_mp4
+        have_mp4 = (have_mp4v2_h or have_mp4_h) and (have_mp4v2 or have_mp4)
 
         if not have_mp4:
             raise Exception('Could not find libmp4, libmp4v2 or the libmp4v2 development headers.')
-- 
2.1.0

